openadmet/models/anvil/workflow.py

```diff
@@ -165,6 +191,7 @@ class ProcedureSpec(SpecBase):
     split: SplitSpec
     feat: FeatureSpec
     model: ModelSpec
+    ensemble: EnsembleSpec
```

Comment:
```suggestion
    ensemble: Optional[EnsembleSpec] = None
```

Then you can remove the optional kwarg.

Reply:
Changes made.